### PR TITLE
Summarize

### PR DESCRIPTION
**Pull request by Georgehe4**
Creating the initial summarizer and unit tests.

@lukechurch @devoncarew

### CODE REVIEW

**Comment by Georgehe4** [comment]
I signed it!

File: test/src/summarize_test.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dartpad.summarize_test;
+
+import 'package:dart_pad/src/summarize.dart';
+import 'package:unittest/unittest.dart';
+
+void defineTests() {
+  group('summarization', () {
```
Comment (devoncarew) [inline test/src/summarize_test.dart]: Let's name the group the same as what it's testing - so `summarize` or `Summarizer`.

File: lib/src/summarize.dart [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class Summarizer {
+  
+  String summarize(String input) {
+//    String size;
+//    String packages;
+//    String libraries;
+//    String features;
+//    String complexity;
+    return "foo";
```
Comment (devoncarew) [inline lib/src/summarize.dart]: nit: needs two mores spaces of indent

File: lib/src/summarize.dart [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dart_pad.summarize;
+
+class Summarizer {
```
Comment (devoncarew) [inline lib/src/summarize.dart]: Let's add some brief dartdoc describing the class (or on the library declaration itself).

File: lib/src/summarize.dart (hunk 2 above)
Comment (devoncarew) [inline lib/src/summarize.dart]: or, the line after needs 2 less :)

**Comment by devoncarew** [comment]
Great, lgtm!

**Comment by devoncarew** [comment]
(intremental work towards https://github.com/dart-lang/dart-pad/issues/344)

**Pull request merged**